apps/backoffice/src/components/app-preview/index.tsx

```diff
@@ -0,0 +1,84 @@
+import { Close, Info, InfoOutlined, Title } from "@mui/icons-material";
+import {
+  Box,
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  FormControl,
+  InputLabel,
+  MenuItem,
+  Paper,
+  Select,
+  Stack,
+  Table,
+  TableBody,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Tooltip,
+  Typography
+} from "@mui/material";
+import React, { useEffect, useState } from "react";
+
+type AppPreviewProps = {
+  showPreview: boolean;
+  onClose: () => void;
+};
+
+export const AppPreview = ({ showPreview, onClose }: AppPreviewProps) => {
+  const [isPreviewOpen, setIsPreviewOpen] = useState<boolean>(showPreview);
+
+  /** Handle close `App Preview` dialog */
+  const handleClose = () => {
+    onClose();
+  };
+
+  useEffect(() => {
+    setIsPreviewOpen(showPreview);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [showPreview]);
+
+  return (
+    <>
+      <Dialog open={isPreviewOpen} onClose={handleClose} disableScrollLock>
+        {/* TODO Struttura stack col con 3 box come row */}
+        <Stack
+          flexDirection={"column"}
+          paddingX={3}
+          paddingY={3}
+          sx={{ minWidth: 600, minHeight: 640 }}
+          padding={"32px"}
+          rowGap={"24px"}
+        >
+          <Stack direction={"row"} gap={1}>
+            <Typography fontWeight={700} variant="h6">
+              Anteprima In App IO
+            </Typography>
+            <Tooltip
+              title={"Versione App xy di App IO"}
+              placement="right"
+              arrow
+            >
+              <InfoOutlined color="disabled" />
+            </Tooltip>
+          </Stack>
+          <Box sx={{ with: 536, height: 467, backgroundImage: "url('/img/app_preview_bg.png')", backgroundSize: "cover" }}></Box>
```

Comment:
I think there's a better way to sizing box without have to specify with and height: just let box cover the entire content

Reply:
[adcda83](https://github.com/pagopa/io-services-cms/pull/810/commits/adcda83e85bddb1cace43606fa26056512d1819b)